frontend/public/locales/en/translation.json

```diff
@@ -1584,6 +1584,8 @@
     "No PolicySets found": "No PolicySets found",
     "No PolicySets match the current search filter": "No PolicySets match the current search filter",
     "No proxy": "No proxy",
+    "No release image is available. Follow cluster creation prerequisite documentation to learn how to add release images.": "No release image is available. Follow cluster creation prerequisite documentation to learn how to add release images.",
```

Comment:
this does not seem to be used anywhere

Reply:
Here https://github.com/stolostron/console/pull/2099/files#diff-77f63e21935d0f15393b4c046cd3ba5ec42d3941b0aaa039ee7e2b45f85c9d9cR108